### PR TITLE
69 add help page and about page

### PR DESCRIPTION
**Pull request by kevinpmcc**
did both help and about in this one. added pages controller, about and help views and updated routes as well as adding links to the nav-bar.

### CODE REVIEW

**Comment by kevinpmcc** [comment]
closes #69

File: app/assets/stylesheets/pages.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Pages controller here.
+// They will automatically be included in application.css.
```
Comment (eripheebs) [inline app/assets/stylesheets/pages.scss:2]: i really dont like the fact rails autogenerates these files it's so annoiyng

Comment (eripheebs) [inline app/assets/stylesheets/pages.scss:2]: yagni yagni

**Comment by eripheebs** [comment]
wonderful content in the about 😆 :shipit:

**Pull request merged**